### PR TITLE
update document of tidb-ctl

### PR DESCRIPTION
**Pull request by imtbkcat**
### What is changed, added or deleted? (Required)

Update document of TiDB Control.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [x] v3.1 (TiDB 3.1 versions)
- [x] v3.0 (TiDB 3.0 versions)
- [x] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### CODE REVIEW

File: tidb-control.md [hunk 1]
```diff
@@ -6,11 +6,17 @@ aliases: ['/docs-cn/dev/reference/tools/tidb-control/','/docs-cn/tools/tidb-cont
...
-TiDB Control 是 TiDB 的命令行工具，用于获取 TiDB 状态信息，多用于调试。
+TiDB Control 是 TiDB 的命令行工具，用于获取 TiDB 状态信息，多用于调试。本文介绍了 TiDB Control 的主要功能和各个功能的使用方法。
 
-## 源码编译
+## 获取 TiDB Contral
 
-编译环境要求：[Go](https://golang.org/) Version 1.7 以上
+### 获取二进制程序
+
+对于使用 [TiDB Ansible](/online-deployment-using-ansible.md) 部署的 TiDB 集群，在 TiDB 的安装路径下可以找到 TiDB Contral 的二进制程序 `tidb-ctl`。
```
Comment (yikeke) [inline tidb-control.md:15]:
```suggestion
对于使用 [TiDB Ansible](/online-deployment-using-ansible.md) 部署的 TiDB 集群，在 TiDB 的安装路径下可以找到 TiDB Control 的二进制程序 `tidb-ctl`。
```

File: tidb-control.md [hunk 2]
```diff
@@ -6,11 +6,17 @@ aliases: ['/docs-cn/dev/reference/tools/tidb-control/','/docs-cn/tools/tidb-cont
 
 # TiDB Control 使用说明
 
-TiDB Control 是 TiDB 的命令行工具，用于获取 TiDB 状态信息，多用于调试。
+TiDB Control 是 TiDB 的命令行工具，用于获取 TiDB 状态信息，多用于调试。本文介绍了 TiDB Control 的主要功能和各个功能的使用方法。
 
-## 源码编译
+## 获取 TiDB Contral
```
Comment (yikeke) [inline tidb-control.md:11]:
```suggestion
## 获取 TiDB Control
```

File: tidb-control.md [hunk 3]
```diff
@@ -28,13 +34,28 @@ TiDB Control 是 TiDB 的命令行工具，用于获取 TiDB 状态信息，多
...
+目前，TiDB Control 包含以下子命令。
+
+* `tidb-ctl base64decode` BASE64 解码
+* `tidb-ctl decoder` 用于 KEY 解码
+* `tidb-ctl etcd` 用于操作 etcd
+* `tidb-ctl log` 格式化日志文件，将单行的堆栈信息展开
+* `tidb-ctl mvcc` - MVCC 信息
+* `tidb-ctl region` - Region 信息
+* `tidb-ctl schema` - Schema 信息
+* `tidb-ctl table` - Table 信息
```
Comment (yikeke) [inline tidb-control.md:46]:
```suggestion
* `tidb-ctl base64decode` 用于 BASE64 解码
* `tidb-ctl decoder` 用于 KEY 解码
* `tidb-ctl etcd` 用于操作 etcd
* `tidb-ctl log` 用于格式化日志文件，将单行的堆栈信息展开
* `tidb-ctl mvcc` 用于获取 MVCC 信息
* `tidb-ctl region` 用于获取 Region 信息
* `tidb-ctl schema` 用于获取 Schema 信息
* `tidb-ctl table` 用于获取 Table 信息
```

File: tidb-control.md [hunk 4]
```diff
@@ -28,13 +34,28 @@ TiDB Control 是 TiDB 的命令行工具，用于获取 TiDB 状态信息，多
...
+
 ### 获取帮助
 
 `tidb-ctl -h/--help` 用于获取帮助信息。tidb-ctl 由多层命令组成，tidb-ctl 及其所有子命令都可以通过 `-h/--help` 来获取使用帮助。
 
-### 连接
+以获取 Schema 信息为例：
+
+通过 `tidb-ctl schema -h` 可以获取这个子命令的使用帮助。schema 有两个子命令，in 和 tid。in 用来通过数据库名获取数据库中所有表的表结构，tid 用来通过全数据库唯一的 table_id 获取表的表结构。
```
Comment (yikeke) [inline tidb-control.md:54]:
```suggestion
通过 `tidb-ctl schema -h` 可以获取这个子命令的使用帮助。schema 有两个子命令——in 和 tid。in 用来通过数据库名获取数据库中所有表的表结构，tid 用来通过全数据库唯一的 table_id 获取表的表结构。
```

File: tidb-control.md [hunk 5]
```diff
@@ -109,12 +113,37 @@ TiDB Control 是 TiDB 的命令行工具，用于获取 TiDB 状态信息，多
 
 这里同样做了截断。
 
-如使用的 TiDB 地址不为默认地址和端口，可以使用命令行参数 `--host`, `--port` 选项，如：`tidb-ctl --host 172.16.55.88 --port 8898 schema in mysql -n db`。
+如使用的 TiDB 地址不为默认地址和端口，可以使用命令行参数 `--host`, `--port` 选项，如：`tidb-ctl --host 172.16.55.88 --port 8898 schema in mysql -n db`
```
Comment (yikeke) [inline tidb-control.md:116]:
```suggestion
如使用的 TiDB 地址不为默认地址和端口，可以使用命令行参数 `--host` 和 `--port` 选项，如：`tidb-ctl --host 172.16.55.88 --port 8898 schema in mysql -n db`。
```

File: tidb-control.md [hunk 6]
```diff
@@ -109,12 +113,37 @@ TiDB Control 是 TiDB 的命令行工具，用于获取 TiDB 状态信息，多
...
+  "O": "stats_meta",
+  "L": "stats_meta"
+ },
+ "charset": "utf8mb4",
+ "collate": "utf8mb4_bin",
+  ...
+}
+``` 
+
+同 in 子命令一样，如果使用的 TiDB 地址不是默认的地址和端口，需要通过 `--host`, `--port` 参数指定 TiDB 的地址和端口。
```
Comment (yikeke) [inline tidb-control.md:139]:
```suggestion
同 in 子命令一样，如果使用的 TiDB 地址不是默认的地址和端口，需要通过 `--host` 和 `--port` 参数指定 TiDB 的地址和端口。
```

Comment (imtbkcat) [reply tidb-control.md]: done, PTAL @yikeke

File: tidb-control.md [hunk 7]
```diff
@@ -6,11 +6,17 @@ aliases: ['/docs-cn/dev/reference/tools/tidb-control/','/docs-cn/tools/tidb-cont
...
-TiDB Control 是 TiDB 的命令行工具，用于获取 TiDB 状态信息，多用于调试。
+TiDB Control 是 TiDB 的命令行工具，用于获取 TiDB 状态信息，多用于调试。本文介绍了 TiDB Control 的主要功能和各个功能的使用方法。
 
-## 源码编译
+## 获取 TiDB Control
 
-编译环境要求：[Go](https://golang.org/) Version 1.7 以上
+### 获取二进制程序
+
+对于使用 [TiDB Ansible](/online-deployment-using-ansible.md) 部署的 TiDB 集群，在 TiDB 的安装路径下可以找到 TiDB Control 的二进制程序 `tidb-ctl`。
```
Comment (yikeke) [inline tidb-control.md:15]: 4.0 开始 TiDB Ansible 部署方式会逐渐下架了，有没有 TiUP 部署方式 获取 TiDB Control 二进制程序的方法？ @imtbkcat

File: tidb-control.md [hunk 8]
```diff
@@ -28,13 +34,28 @@ TiDB Control 是 TiDB 的命令行工具，用于获取 TiDB 状态信息，多
 * -n: 选项
 * db: -n 的参数
 
+目前，TiDB Control 包含以下子命令。
+
+* `tidb-ctl base64decode` BASE64 解码
```
Comment (yikeke) [inline tidb-control.md:39]:
```suggestion
* `tidb-ctl base64decode` 用于 BASE64 解码
```
这里加上「用于」比较像句子？

File: tidb-control.md [hunk 9]
```diff
@@ -6,11 +6,17 @@ aliases: ['/docs-cn/dev/reference/tools/tidb-control/','/docs-cn/tools/tidb-cont
 
 # TiDB Control 使用说明
 
-TiDB Control 是 TiDB 的命令行工具，用于获取 TiDB 状态信息，多用于调试。
+TiDB Control 是 TiDB 的命令行工具，用于获取 TiDB 状态信息，多用于调试。本文介绍了 TiDB Control 的主要功能和各个功能的使用方法。
 
-## 源码编译
+## 获取 TiDB Control
```
Comment (yikeke) [inline tidb-control.md:11]:
```suggestion
## 获取 TiDB Control

本节提供了两种方式获取 TiDB Control 工具。
```

File: tidb-control.md (hunk 7 above)
Comment (yikeke) [reply tidb-control.md:15]: 同 https://github.com/pingcap/docs-cn/pull/3189#pullrequestreview-416615919

**Comment by imtbkcat** [comment]
PTAL @kissmydb @yikeke

File: tidb-control.md [hunk 10]
```diff
@@ -6,11 +6,19 @@ aliases: ['/docs-cn/dev/reference/tools/tidb-control/','/docs-cn/tools/tidb-cont
...
 
-编译环境要求：[Go](https://golang.org/) Version 1.7 以上
+本节提供了两种方式获取 TiDB Control 工具。
+
+### 通过 TiUP 安装
+
+在安装 TiUP 之后， 可以使用 `tiup ctl tidb` 获取 TiDB Control 的二进制程序。通过执行 `tiup ctl tidb` 来运行 TiDB Control。
```
Comment (yikeke) [inline tidb-control.md:17]:
```suggestion
在安装 TiUP 之后， 可以使用 `tiup ctl tidb` 命令来获取 TiDB Control 的二进制程序以及运行 TiDB Control。
```
To make it concise

**Comment by yikeke** [comment]
注意：现在这篇文档提供了 TiUP 内容（只适用于 master 和 v4.0），到时候 cherry-pick 这个 PR 到 v3.1、v3.0、v2.1 分支的时候，还是得换成 Ansible 的内容。FYI @imtbkcat

**Comment by jackysp** [comment]
@imtbkcat please address the comments.

**Comment by imtbkcat** [comment]
PTAL @kissmydb

**Pull request merged**